backend/grants/tasks.py

```diff
@@ -31,20 +31,14 @@ def send_grant_reply_approved_email(*, grant_id, is_reminder):
         "Reminder: Financial Aid Update" if is_reminder else "Financial Aid Update"
     )
 
-    if not grant.conference.visa_application_form_link:
```

Comment:
Maybe I should add a check if there is the '/visa' page in the website? 
In case we forgot to add the page next year...